lib/FitAnalyticsWidget.js

```diff
@@ -210,7 +224,7 @@ class FitAnalyticsWidget extends Component {
   render() {
     return <WebView
       ref={this.webview}
-      source={{ uri: 'https://widget.fitanalytics.com/widget/app-embed.html' }}
+      source={{ uri: 'https://widget.fitanalytics.com/widget/app-embed.html?rn=0.7.0' }}
```

Comment:
What does `rn` stands for?

Reply:
Reverted, the cache-busting will be implement by other means.